docs/source/staking/running_a_worker.rst

```diff
@@ -227,6 +227,17 @@ Replace the following values with your own:
    * ``<GWEI>`` - The maximum price of gas to spend on commitment transactions
 
 
+The configuration settings will be stored in an ursula configuration file.
+
+.. important::
+
+    The default configuration file is ``ursula.json``. If there is an existing default configuration file, a new
+    Worker configuration file suffixed by the first 8 characters of the node's signing public key
```

Comment:
For simplification perhaps just use the wording "the node's public key" - We can start to suggest a convention, afterall this is the core element and invariant of the node's identity.

Reply:
✔️